trackintel/model/positionfixes.py

```diff
@@ -54,19 +56,32 @@ def _validate(obj):
                 % (", ".join(PositionfixesAccessor.required_columns), ", ".join(obj.columns))
             )
 
+        geom_isna = obj.geometry.isna()
+
         # check geometry
-        assert obj.geometry.is_valid.all(), (
-            "Not all geometries are valid. Try x[~ x.geometry.is_valid] " "where x is you GeoDataFrame"
-        )
+        assert (
+            geom_isna | obj.geometry.is_valid
+        ).all(), "Not all geometries are valid. Try x[~ x.geometry.is_valid] where x is you GeoDataFrame"
 
-        if obj.geometry.iloc[0].geom_type != "Point":
-            raise AttributeError("The geometry must be a Point (only first checked).")
+        if "Point" not in obj.geometry.geom_type.values:
+            raise AttributeError("The geometry must be a Point (checked for any occurrence).")
```

Comment:
nice idea to change this :) I think it would have to be `assert all(obj.geometry.geom_type.values == "Point")`, right? 
Or probably we would have to consider the missing values again? I just tested it with some other data, seems like the type of an empty point is GeometryCollection, so it would have to be 
```
assert all(obj.geometry.geom_type.isin(["Point", None, "GeometryCollection"]))
```